.github/workflows/build-images-release.yaml

```diff
@@ -0,0 +1,148 @@
+name: Image Release Build
```

Comment:
Reviewer note: This file is basically a copy of https://github.com/cilium/cilium/blob/master/.github/workflows/build-images-releases.yaml

Diff:
```diff
--- ./cilium/.github/workflows/build-images-releases.yaml	2021-08-02 14:29:11.413432217 +0200
+++ ./certgen/.github/workflows/build-images-release.yaml	2021-08-02 16:35:08.166602836 +0200
@@ -10,38 +10,15 @@
 
 jobs:
   build-and-push:
-    if: ${{ github.repository == 'cilium/cilium' }}
+    if: ${{ github.repository == 'cilium/certgen' }}
     environment: release
     runs-on: ubuntu-20.04
     strategy:
+      # matrix for easier code-sharing with workflows from cilium/cilium
       matrix:
         include:
-          - name: cilium
-            dockerfile: ./images/cilium/Dockerfile
-
-          - name: operator
-            dockerfile: ./images/operator/Dockerfile
-
-          - name: operator-aws
-            dockerfile: ./images/operator/Dockerfile
-
-          - name: operator-azure
-            dockerfile: ./images/operator/Dockerfile
-
-          - name: operator-alibabacloud
-            dockerfile: ./images/operator/Dockerfile
-
-          - name: operator-generic
-            dockerfile: ./images/operator/Dockerfile
-
-          - name: hubble-relay
-            dockerfile: ./images/hubble-relay/Dockerfile
-
-          - name: clustermesh-apiserver
-            dockerfile: ./images/clustermesh-apiserver/Dockerfile
-
-          - name: docker-plugin
-            dockerfile: ./images/cilium-docker-plugin/Dockerfile
+          - name: certgen
+            dockerfile: ./Dockerfile
 
     steps:
       - name: Set up Docker Buildx
@@ -82,8 +59,6 @@
             ${{ github.repository_owner }}/${{ matrix.name }}:${{ steps.tag.outputs.tag }}
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}:${{ steps.tag.outputs.tag }}
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:${{ github.sha }}
-          build-args: |
-            OPERATOR_VARIANT=${{ matrix.name }}
 
       - name: Image Release Digest
         shell: bash
@@ -118,7 +93,7 @@
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
 
   image-digests:
-    if: ${{ github.repository == 'cilium/cilium' }}
+    if: ${{ github.repository == 'cilium/certgen' }}
     name: Display Digests
     runs-on: ubuntu-20.04
     needs: build-and-push
@@ -149,7 +124,7 @@
         shell: bash
         run: |
           cd image-digest/
-          echo "# File generated by .github/workflows/images-legacy-releases.yaml; DO NOT EDIT." > ../Makefile.digests
+          echo "# File generated by .github/workflows/build-images-releases.yaml; DO NOT EDIT." > ../Makefile.digests
           echo "# Copyright "$(date +'%Y')" Authors of Cilium" >> ../Makefile.digests
           echo "# SPDX-License-Identifier: Apache-2.0" >> ../Makefile.digests
           echo "" >> ../Makefile.digests
```